python/raf/distributed/op.py

```diff
@@ -193,32 +193,73 @@ def broadcast(x, root):
     return sym._broadcast(x, root)
 
 
-def all_to_all(x, group_use_memcpy=False):
+def all_to_all(x, split_axis=0, concat_axis=0, rank_list=None, group_use_memcpy=False):
```

Comment:
It seems that the version of all_to_all only receives a single tensor as input. If that's the case, should we also remove the unused "group_use_memcpy" parameter?

Reply:
I think neither in normal MoE models nor in models optimized by our previous work, there's no all_to_all that receives multiple tensors as input. In normal MoE models, all_to_alls depend on each other. In optimized models, there could be all_to_alls that can be executed in parallel, but fusing them would break the pipeline.

So I think we can safely neglect this case and remove the feature of `group_use_memcpy`.